services/QuillLMS/app/controllers/cms/users_controller.rb

```diff
@@ -147,12 +147,12 @@ def complete_sales_stage
   end
 
   protected def user_params
-    params.require(:user).permit([:name, :email, :username, :title, :role, :classcode, :password, :password_confirmation, :flags =>[]] + default_params
+    params.require(:user).permit([:name, :email, :flagset, :username, :title, :role, :classcode, :password, :password_confirmation, :flags =>[]] + default_params
     )
   end
 
   protected def user_query_params
-    params.permit(@text_search_inputs.map(&:to_sym) + default_params + [:page, :user_role, :user_flag, :sort, :sort_direction, :user_premium_status, :class_code])
+    params.permit(@text_search_inputs.map(&:to_sym) + default_params + [:flagset, :page, :user_role, :user_flag, :sort, :sort_direction, :user_premium_status, :class_code])
```

Comment:
If we're no longer using `user_flag` below, should we pull that value out of the list of params here?